waspc/src/Wasp/AI/GenerateNewProject/Page.hs

```diff
@@ -150,20 +152,26 @@ pageDocPrompt =
               <div className="p-4 bg-slate-50">
                 {user.username}
                 {task.id}
-                <button onClick={createTask({description: 'new task'})} className="bg-slate-500 hover:bg-slate-700 text-white font-bold py-2 px-4 rounded"> + </button>
+                <button onClick={createTaskFn({description: 'new task'})} className="bg-slate-500 hover:bg-slate-700 text-white font-bold py-2 px-4 rounded"> + </button>
                 <button onClick={logout} className="bg-slate-500 hover:bg-slate-700 text-white font-bold py-2 px-4 rounded"> Logout </button>
               </div>
             );
           };
         ```
 
-        A bit about importing actions and queries.
-        If query is named "myQuery", its import should look like `import myQuery from '@wasp/queries/myQuery';`.
-        It has to be default import, and name of the file is the same as name of the query.
-        Same rules apply for actions.
-        One import statement per query/action.
+        Here are the rules for importing actions and queries.
 
-        Note: there is no `useMutation` or `useAction` hook to import in Wasp. Actions are to be called directly.
+        If a query is called "myQuery", its import should look like `import myQuery from '@wasp/queries/myQuery';`.
+        It has to be a default import, and name of the file is the same as name of the query.
+        The hook for wrapping queries is called `useQuery`.
+        Use a single import statement per query.
+
+        If an action is called "myAction", its import should look like `import myAction from '@wasp/actions/myAction';`.
```

Comment:
I wonder if you could just say, instead of repeating the whole thing, smth like `It is analogous for actions`, like we had before, I think that worked? We save a bit of tokens.

Reply:
I could try, but this is probably safer. Is the token difference significant?

Reply:
Tried it, it doesn't really work. It does this:
```typescript
import { useQuery, useAction } from '@wasp/queries';
```